alvr/xtask/src/main.rs

```diff
@@ -52,6 +52,7 @@ FLAGS:
     --no-rebuild        Do not rebuild the streamer with run-streamer
     --ci                Do some CI related tweaks. Depends on the other flags and subcommand
     --no-stdcpp         Disable linking to libc++_shared with build-client-lib
+    --all-targets       Use all targets for prepare-deps and build-client-lib subcommand
```

Comment:
Can we elaborate what the flag does like, "For prepare-deps and build-client-lib subcommand, will build for all android supported ABI targets". will be useful in future docs.

Reply:
Done